src/pages/getting-started/app-guides/chat-actions/index.mdx

```diff
@@ -8,11 +8,11 @@ desc: "Create custom Chat Actions allowing agents to speed up common tasks."
 ---
 
 <p>
-Chat Actions are one of the easiest way to extend the LiveChat Agent Application. Chat Actions are accessible from the contextual menu (represented by <div style={{display: 'inline-block', width: '18px'}}><img style={{verticalAlign: 'middle'}} src="/images/getting-started/app-guides/chat-actions/menu-icon.svg" alt="meatballs-menu-icon" height="18px" width="18px" /></div>) in the Agent Application. Use them to help agents speed up their daily work by automating common tasks, such as:
+Chat Actions are one of the easiest ways to extend the LiveChat Agent Application. Chat Actions are accessible from the contextual menu (represented by <div style={{display: 'inline-block', width: '18px'}}><img style={{verticalAlign: 'middle'}} src="/images/getting-started/app-guides/chat-actions/menu-icon.svg" alt="meatballs-menu-icon" height="18px" width="18px" /></div>) in the Agent Application. Use them to help agents speed up their daily work by automating common tasks, such as:
 </p>
 
 - opening a link to an external CRM
-- displaying a modal with a form that allows to create a new issue, task, or contact
+- displaying a modal with a form that allows to create a new issue, task or contact
```

Comment:
I would keep that come here as both options are correct. However, from experience and research, Americans would use it more often, and we try to follow American English:)